components/pedigree/resources/src/main/resources/pedigree/hpoTerm.js

```diff
@@ -75,7 +75,7 @@ define([
     }
 
     HPOTerm.getServiceURL = function() {
```

Comment:
I would remove this and added `getHPOSuggestURL()` (or smth similar) to `externalEndpoints.js`